js/common/charts.d.ts

```diff
@@ -64,6 +64,12 @@ export type HatchDirection = 'left' | 'none' | 'right';
  */
 export type LabelOverlap = 'hide' | 'none';
 
+/**
+ * @public
+ * @namespace DevExpress.common.charts
+ */
+export type ShiftLabelOverlap = 'hide' | 'none' | 'shift';
```

Comment:
@vladaskorohodova are we OK with this name?
As for me there is something confusing in how the word 'shift' used: maybe 'shiftable' or 'shifted' fit better?
Smth like `ShiftableLabelOverlap` or `LabelShiftedOverlap`

Reply:
After discussing with Vlada, we left the name ShiftLabelOverlap